spec/src/main/asciidoc/microprofile-openapi-spec.adoc

```diff
@@ -186,35 +227,225 @@ public Response findPetsByStatus(...) { ... }
     operationId: findPetsByStatus
 ----
 
+.Sample 2 - Operation with different responses
+[source,Java]
+----
+@GET
+@Path("/{username}")
+@Operation(summary = "Get user by user name",
+        responses = {
+                @APIResponse(description = "The user",
+                        content = @Content(mediaType = "application/json",
+                                schema = @Schema(implementation = User.class))),
```

Comment:
Inconsistent tabbing of the previous 3 lines - the `description` and `content` and `schema` attributes should line up.

Reply:
+1